airbyte-webapp/scripts/regenerate-api-clients.ts

```diff
@@ -0,0 +1,17 @@
+import { statSync, existsSync } from "fs";
+import { resolve, basename } from "path";
+
+import chalk from "chalk";
+
+import conf from "../orval.config";
+
+const lastTouched = (filepath: string) => statSync(resolve(filepath)).mtime;
+
+for (const api of Object.values(conf)) {
+  if (!existsSync(api.output.target) || lastTouched(api.input) > lastTouched(api.output.target)) {
+    // no need to read and parse the orval lib if all files are up to date
+    require("orval").generate(api);
+  } else {
+    console.log(`🎉 ${chalk.green(basename(api.output.target))} is up-to-date`);
```

Comment:
@ambirdsall Added some output if a file is up-to-date (mirroring the output of Orval), to make sure it's still visible that something was checked.